src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class Lottos(val lottoList: List<Lotto>) {
+    val buyCount: Long
+        get() = lottoList.size.toLong()
+
+    companion object {
+        fun from(money: Money): Lottos {
+            val buyCount = money.buy(Lotto.PRICE)
+            val lottoList = buildList {
+                repeat((1..buyCount).count()) {
+                    add(Lotto())
+                }
+            }
```

Comment:
```suggestion
val buyCount = money.buy(Lotto.PRICE)
            val lottoList = buildList {
                repeat(buyCount) {
                    add(Lotto())
                }
            }
```
buyCount가 Int를 반환한다면 repeat 로직이 좀 더 깔끔해 질 것 같네요 😊